src/components/Text.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled";
+
+const StyledP = styled.p`
```

Comment:
`StyledText`. `StyledP` is somewhat opaque since `<p />` tags are applicable to a wide range of components.